src/js/common/components/FAQBody.jsx

```diff
@@ -135,18 +173,18 @@ export default class FAQBody extends Component {
             url="https://github.com/WeVote"
             target="_blank"
             body="100+ contributors on GitHub."
+            trackingOn
           />
         </Suspense>
         We also have
         {' '}
-        <Suspense fallback={<></>}>
-          <OpenExternalWebSite
-            linkIdAttribute="weVoteAboutUsPage"
-            url={`${webAppConfig.WE_VOTE_URL_PROTOCOL + webAppConfig.WE_VOTE_HOSTNAME}/more/about`}
-            target="_blank"
-            body="volunteer board members"
-          />
-        </Suspense>
+        <Link
```

Comment:
The code before you changed it has a bug.  This should be a historyPush, otherwise by linking to wevote as if it were an external site, will cause the session to be destroyed and recreated, potentially losing data. 
So in this case at this line you should call a onCick function like 
`routeToAbout () {
    this.pushDataLayer('/more/about', 'faq-more-about')
    historyPush(''/more/about');
}
`